src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesDeclarativeAgentTest.java

```diff
@@ -169,4 +169,12 @@ public void declarativeWithNonexistentDockerImage() throws Exception {
         r.assertBuildStatus(Result.FAILURE, r.waitForCompletion(b));
         r.assertLogContains("ERROR: Unable to pull Docker image", b);
     }
+
+    @Issue("JENKINS-61360")
+    @Test
+    public void declarativeShowRawYamlFalse() throws Exception {
+        assertNotNull(createJobThenScheduleRun());
+        r.assertBuildStatusSuccess(r.waitForCompletion(b));
+        r.assertLogContains("CONTAINER_ENV_VAR = jnlp\n", b);
```

Comment:
Need something like
```
r.assertLogNotContains("class: KubernetesDeclarativeAgentTest", b);
```

Reply:
Thanks for the correction Vincent, the test validation was not correct.
I have already modified the method.